xmtp_api_http/src/lib.rs

```diff
@@ -50,6 +55,43 @@ impl XmtpHttpApiClient {
     }
 }
 
+impl ClientWithMetadata for XmtpHttpApiClient {
+    fn set_app_version(&mut self, version: String) -> Result<(), Error> {
+        self.app_version = Some(version);
+
+        let mut headers = header::HeaderMap::new();
+        if let Some(app_version) = &self.app_version {
+            headers.insert("x-app-version", app_version.parse().unwrap());
+        }
+        if let Some(libxmtp_version) = &self.libxmtp_version {
+            headers.insert("x-libxmtp-version", libxmtp_version.parse().unwrap());
```

Comment:
probalby don't want to be unwrapping -- could default or error

Reply:
Ya. That's a good call out